packages/dotcom-ui-footer/src/components/partials.tsx

```diff
@@ -63,31 +63,27 @@ const FooterContents = ({ footerData }: TFooterContentsProps) => (
         // The Next navigation API splits footer links data into "columns"
         // <https://github.com/Financial-Times/next-navigation-api/blob/HEAD/server/transforms/footer.js>
         const submenu = item.submenu.items as TNavMenuItem[][]
-
         return (
-          <div
-            key={`group-${index}`}
-            className={`o-footer__matrix-group o-footer__matrix-group--${submenu.length}`}>
+          <div key={`group-${index}`} className={`o-footer__matrix-group o-footer__matrix-group--1`}>
             <SectionTitle label={item.label} submenu={submenu} index={index} />
             <SectionLinks submenu={submenu} index={index} />
           </div>
         )
       })}
+      <div className="o-footer__matrix-group o-footer__matrix-group--1">
+        <h3 className="o-footer__matrix-title o-footer__matrix-title--external-link">
+          <a
+            className="o-footer__matrix-link o-footer__matrix-link--more"
+            id="o-footer-section-5"
```

Comment:
This id should be generated programmatically I think, as we don't know for sure it is number 5.
I think it should be `o-footer-section-${footerData.length}`